package.json

```diff
@@ -41,7 +41,9 @@
     "db:migrate:undo": "node_modules/.bin/sequelize db:migrate:undo",
     "db:migrate:undo:prod": "node_modules/.bin/sequelize db:migrate:undo:all --options-path .production.sequelizerc",
     "db:seed": "node_modules/.bin/sequelize db:seed:all",
-    "db:seed:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:seed:all",
+    "db:seed:test": "node_modules/.bin/sequelize db:seed --seed 20201124160449-users 20201209172017-approvers 20201210172017-grantees 20201211172017-grants 20210127161802-add-goals 20210409213554-cdi-grants",
+    "db:seed:ci": "cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:seed --seed 20201124160449-users 20201209172017-approvers 20201210172017-grantees 20201211172017-grants 20210127161802-add-goals 20210409213554-cdi-grants",
+    "db:seed:axe": "yarn db:seed:ci && cross-env POSTGRES_USERNAME=postgres POSTGRES_DB=ttasmarthub node_modules/.bin/sequelize db:seed --seed 20210810202038-approved-report",
```

Comment:
Can we add the activity report to all seeders and drop the axe specific `db:seed:axe`?

Reply:
We can, but I was trying to avoid polluting the unit test data. If that is not an issue, then this could certainly be simplified.

Reply:
My preference would be to have a single set of seeders. Although if it causes a bunch of tests to fail the juice probably isn't worth the squeeze.

Reply:
In my first run about 10 unit tests failed, so I could go the route of changing those tests instead. My concern was more that there would be a report floating around that we'd have to be aware of as we write new backend tests.

Reply:
Unfortunately we already have to worry about floating reports (if they are created in the seeders or not) because all the tests run in parallel and some create activity reports.